webapp/src/components/kanban/kanban.tsx

```diff
@@ -28,159 +32,52 @@ type Props = {
-                                <KanbanCard
-                                    card={card}
-                                    visiblePropertyTemplates={visiblePropertyTemplates}
-                                    key={card.id}
-                                    readonly={this.props.readonly}
-                                    isSelected={this.props.selectedCardIds.includes(card.id)}
-                                    onClick={(e) => {
-                                        this.props.onCardClicked(e, card)
-                                    }}
-                                    onDrop={this.onDropToCard}
-                                    showCard={this.props.showCard}
-                                    isManualSort={isManualSort}
-                                />
-                            ))}
-                            {!this.props.readonly &&
-                                <Button
-                                    onClick={() => {
-                                        this.props.addCard(group.option.id, true)
-                                    }}
-                                >
-                                    <FormattedMessage
-                                        id='BoardComponent.new'
-                                        defaultMessage='+ New'
-                                    />
-                                </Button>
-                            }
-                        </KanbanColumn>
-                    ))}
-
-                    {/* Hidden columns */}
-
-                    {hiddenGroups.length > 0 &&
-                        <div className='octo-board-column narrow'>
-                            {hiddenGroups.map((group) => (
-                                <KanbanHiddenColumnItem
-                                    key={group.option.id}
-                                    group={group}
-                                    boardTree={boardTree}
-                                    intl={this.props.intl}
-                                    readonly={this.props.readonly}
-                                    onDrop={(card: Card) => this.onDropToColumn(group.option, card)}
-                                />
-                            ))}
-                        </div>}
-                </div>
-            </div>
-        )
-    }
-
-    private propertyNameChanged = async (option: IPropertyOption, text: string): Promise<void> => {
-        const {boardTree} = this.props
+    const propertyValues = groupByProperty.options || []
+    Utils.log(`${propertyValues.length} propertyValues`)
+
+    const {board, activeView, visibleGroups, hiddenGroups} = boardTree
+    const visiblePropertyTemplates = board.cardProperties.filter((template) => activeView.visiblePropertyIds.includes(template.id))
+    const isManualSort = activeView.sortOptions.length < 1
+
+    const [cardTrees, setCardTrees] = useState<{[key: string]: CardTree | undefined}>({a: undefined})
```

Comment:
since it can be undefined, would it not work to just do `()` rather than do `{a: undefined}` ? 
```suggestion
    const [cardTrees, setCardTrees] = useState<{[key: string]: CardTree | undefined}>()
```
This wouldn't work?

Reply:
Hey, yes This was added for debugging and I missed removing it.